File: trunk/net.tenteCsharp.templatesProject/src_code/lightManagement/DeviceKind.java
cclass lightManagement.LightManagement;

public cclass DeviceKind extends TypeEnum {
	
	public DeviceKind (){
		super();		
		values.add("LightSwitch");		
		values.add("LightDimmer");		
		values.add("LightController");
	}
	
}

File: trunk/net.tenteCsharp.templatesProject/src-gen/lightManagement/DeviceKind.cs
using System;
using System.Collections;
using System.Text;

namespace SmartHome
{
	/// <summary>
    /// Enumeration
    /// </summary>
	public class DeviceKind : TypeEnum 
	{
		public DeviceKind () 
			: base()
		{		
		values.Add("LightSwitch");		
		values.Add("LightDimmer");		
		values.Add("LightController");
		}
	
	}
}